client/components/App.jsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-import ProductOverview from './ProductOverview/ProductOverview';
+import RelatedProducts from './relatedProducts/RelatedProducts';
 
-const App = () => (
-  <>
-    <div>Header</div>
-    <div><ProductOverview /></div>
-  </>
-);
+import '../css/App.scss';
+
+const App = () => {
+  // const [products, setProducts] = useState([]);
+  const [currentProduct, setCurrentProduct] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    axios.get('/api/products')
+      .then(({ data }) => {
+        // setProducts(data);
+        setCurrentProduct(data[0]);
+        setIsLoading(false);
+      });
```

Comment:
Would you like to add an error catch for the promise?

Reply:
@runhui2010 good suggestions. @cschucode good job and see my comments with required changes, follow up tickets and possible exploration areas.

Reply:
Added minimal error message for now.